### PR TITLE
Request #4: Feature contact form

### PR DESCRIPTION
**Pull request by gabyse1**
This branch contains the design of the contact form feature and the bottom navigation of my portfolio.
1. Contact form:
    - Insert a full name input control that constrains the maximum number of characters to 30.
    - Insert an email input control that constrains the type to correct email addresses.
    - Insert a message Textarea control that constrains the maximum number of characters to 500.
    - Collect data by using the Formspree service.
2. Bottom navigation:
    - Insert a list that displays a list of media social icons with a different style than the headline section.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -510,3 +529,85 @@ header {
...
+  width: calc(100% - 24px);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  margin: 100px 24px;
+}
+
+.contact-form .form-description {
+  font-family: 'Roboto', sans-serif;
```
Comment (leolpaz) [inline css/styles.css]: - [ ] To be more in line with the Figma design, kindly use the same font for your form and form elements, in the case of your template, it seems to be Inter

image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @gabyse1 , 

You've done an amazing job so far, I'd like to highlight the following:
- Great job with semantic HTML
- Great job with using display flex
- Good job with the repo

One issue needs to be resolved before your approval, but you're almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 1 above)
Comment (gabyse1) [reply css/styles.css:551]: I am going to change it.

**Review by Gambit142 — APPROVED** [review]
Hi @gabyse1 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**